### ISSUE TITLE
[Feature] Please pull data from webcompat

### ISSUE THREAD

**Issue by andreicristianpetcu** [webhintio/hint#2600]
# 🚀 Feature request

## Description
Please pull data from https://webcompat.com/ or their [github repo](https://github.com/webcompat/web-bugs/issues)

 What do you want to be added?
If I scan google.com I would like pull all the issues reported there. For example [these issues](https://github.com/webcompat/web-bugs/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+in%3Atitle+%27google.com+-+%27)
![image](https://user-images.githubusercontent.com/922368/59555642-538fc500-8fbe-11e9-831d-71d347039fea.png)

 Are you willing to work on this?
No

## What scenarios will this solve?
I scan my own site with this tool to see various issues it has. If somebody reported an issue on my site, I would like to see that issue as a link.

 Why do you want/need this?
Both hint and webcompat seem to want to reduce web compatibility issues. While hint basically does static analysis, it can fetch data from other sources like this repo for compat bug reports.

**Comment by antross** [comment]
Thanks @andreicristianpetcu for the great suggestion!

This definitely seems like something that could help provide deeper feedback beyond what can be automatically identified. It seems like it would make sense as a new hint under the Compatibility category.

Though I don't see a dedicated API, I see this could be queried via GitHub's API or we could consider taking periodic snapshots of the essential parts of the data (likely URL, browser, and issue number) to flag issues and link out to the site and/or repo for more information.

**Comment by foolip** [comment]
cc webcompat.com folks @miketaylr @karlcow @adamopenweb

My own quick read of the situation is that https://github.com/webcompat/web-bugs/issues aren't broken down by the APIs involved or root cause, so there isn't a ready data source to plug in. It's possible, however, that text analysis of the comments of all issues could yield some useful statistics.

**Comment by miketaylr** [comment]
Correct, there's no API or nice breakdown of "these APIs break these sites" -- but you could at least point to open issues for a given domain.
